docs/meshstack.billing.md

```diff
@@ -239,7 +239,31 @@ The following configuration options are available at `mesh.kraken.productcatalog
 
 Each project in meshStack is associated with a Chargeback Account. meshStack periodically generates [chargeback statements](meshcloud.project-metering.md#chargeback-statements).
 
-The attributes that shall be part of the billing info on the chargeback statements can be configured as follows.
+The attributes that shall be part of the billing info on the chargeback statements can be configured.
+
+### Available metadata keys
```

Comment:
Isn't this very customer specific? The metadata available is very specific to a meshStack

Reply:
The meshStack specific part is under `Available tag keys`